src/partials/annotations.editor.html

```diff
@@ -0,0 +1,29 @@
+<div class="gf-form-group">
+  <div class="gf-form-inline">
+    <div class="gf-form gf-form--grow">
+      <input
+       class="gf-form-input"
+       placeholder="query expression"
+       ng-model="ctrl.annotation.queryText"
+       ></input>
+    </div>
+  </div>
+</div>
+
+<div class="gf-form-group">
+	<h6>Field mappings</h6>
+	<div class="gf-form-inline">
+		<div class="gf-form">
+			<span class="gf-form-label">Time</span>
+			<input type="text" class="gf-form-input max-width-14" ng-model='ctrl.annotation.timeField' placeholder="timestamp"></input>
+		</div>
+		<div class="gf-form">
+			<span class="gf-form-label">Time End</span>
```

Comment:
If my understanding is correct, the only time they would have a `timeEnd` field is if they have a special event that contains data for a region annotation - `time` and `timeEnd`.  Unless they have something like this, in most cases we will just use the event `timestamp` field.

Reply:
That is correct, `timeEnd` defaults to nothing and isn't put into the annotation object at all in that case. Default for `time` is `timestamp`, and we end up with just a vertical line for the annotation.

Reply:
We can leave this as is, but is sounds like to me for 80-90+% of the cases, the user won't have have a `timeEnd`.  If we don't support `timeEnd`, we could simplify things to where they just need the `message` field and we default to the `timestamp` field for the `time.